docs/mast/mast_mastquery.rst

```diff
@@ -3,26 +3,118 @@
 MAST Queries
 ************
 
-Direct Mast Queries
-===================
-
 The Mast class provides more direct access to the MAST interface.  It requires
 more knowledge of the inner workings of the MAST API, and should be rarely
 needed.  However in the case of new functionality not yet implemented in
 astroquery, this class does allow access.  See the `MAST api documentation
-<https://mast.stsci.edu/api>`_ for more information.
+<https://mast.stsci.edu/api>`__ for more information.
+
+The basic MAST query function accepts
+`MAST Service <https://mast.stsci.edu/api/v0/_services.html>`__-specific
+parameters and returns query results as an `~astropy.table.Table`.
+
+Filtered Mast Queries
+=====================
+
+MAST's Filtered services use the parameters 'columns' and 'filters'. The 'columns'
+parameter is a required string that specifies the columns to be returned as a
+comma-separated list. The 'filters' parameter is a required list of filters to be
+applied. The `~astroquery.mast.MastClass.mast_query` method accepts that list of
+filters as keyword arguments paired with a list of values.
+
+.. doctest-remote-data::
+
+   >>> from astroquery.mast import Mast
+   ...
+   >>> observations = Mast.mast_query('Mast.Jwst.Filtered.Nirspec',
+   ...                                columns = 'title, instrume, targname',
```

Comment:
Where can people go to see what options they have for `filters` and `columns`? i.e. How would a user know that the column name is `instrume` and not `instrument`? Be sure to post a resource pointing to this.